src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -215,6 +217,35 @@ public KaitaiGenerator opaqueTypes(Boolean opaqueTypes) {
 		return this;
 	}
 
+	/**
+	 * Get version check mode.
+	 *
+	 * @return Version check mode
+	 */
+	public Boolean isNoVersionCheck() {
```

Comment:
Prefix `is` on getters preffered for primitive boolean fields because it's is nonnullable, but for Object-backed fields is preferred `get` prefix.

Reply:
Moved to a primitive type